content/events/hacktoberfest.adoc

```diff
@@ -9,50 +9,46 @@ tags:
   - programs
   - hacktoberfest
 links:
-  discourse: https://community.jenkins.io/t/hacktoberfest-2022/3805
+  discourse: https://community.jenkins.io/t/hacktoberfest-2023/9613
 opengraph:
-  image: /images/hacktoberfest/hacktoberfest_2022_social.png
+  image: /images/hacktoberfest/hacktoberfest_2023_social_600x300.png
 ---
 
-image:/images/hacktoberfest/hacktoberfest_2022.svg[Hacktoberfest, role=center]
+image:/images/hacktoberfest/hacktoberfest_2023_logo.png[Hacktoberfest, role=center]
 
 link:https://hacktoberfest.com/[Hacktoberfest]
-is a month long celebration of open source software.
-It happens every year in October.
-During this event everyone can support open-source by contributing changes, and then earn a limited edition swag.
-
-We invite everyone to participate and to contribute to the Jenkins project,
-regardless of their background and Jenkins experience.
-There are many ways to
-link:/participate/[contribute] to Jenkins during Hacktoberfest.
-You can work on code or documentation,
-contribute to link:https://wiki.jenkins.io/display/JENKINS/Internationalization[localization],
-create new link:/artwork[artwork], etc.
-You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
-Generally, any pull requests in GitHub would qualify.
-See the link:/participate/[Contribute and Participate] page for more information about how to contribute.
+is a month-long celebration of open source software that happens every October.
+During this event, everyone can support open source by contributing changes and earning a unique digital reward.
+This year marks the 10th anniversary of Hacktoberfest!
 
+We invite everyone to participate and contribute to the Jenkins project, regardless of their background and Jenkins experience.
+There are many ways to link:/participate/[contribute] to Jenkins during Hacktoberfest.
+You can participate by working on link:https:/participate/code/[code] or link:/participate/document/[documentation], contributing to link:/doc/developer/internationalization/[localization], or creating new link:/artwork[artwork].
+You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
+Generally, any four non-spammy pull requests in GitHub or merge requests in GitLab would qualify.
```

Comment:
Since the Jenkins project has no presence on GitLab.com, I think we should consider only mentioning GitHub.
```suggestion
Generally, any four non-spammy pull requests in GitHub would qualify.
```